package.json

```diff
@@ -32,7 +32,6 @@
     "@ucanto/validator": "^9.0.0",
     "@web-std/blob": "^3.0.4",
     "@web-std/fetch": "^4.1.0",
-    "@web3-storage/capabilities": "^9.3.0",
```

Comment:
I can't see any code at the top level that depends directly on capabilities, so this is removed rather than updated.